doc/source/examples/multiphysics/laser-meltpool/laser-meltpool.rst

```diff
@@ -176,41 +183,52 @@ The laser heat source locally melts the material, which is initially in the soli
     # Physical Properties
     #---------------------------------------------------
     subsection physical properties
-      set number of fluids     		    = 1
-      subsection fluid 0
-        set thermal conductivity model 	= phase_change
-        set rheological model 		    = phase_change
-        set specific heat model 		    = phase_change
+      set number of fluids                      = 2
```

Comment:
If the VOF is disable, should the `number of fluids  = 1` ?

Reply:
True, it is revised.